public/index.html

```diff
@@ -15,6 +15,12 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link
+    rel="stylesheet"
+    href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"
```

Comment:
I see we have now thee libraries for styling 
- bootstap 
- @emotion/styled
- @mui/material

Note that using a lot of third party libraries will increase code size. 

so better to use only one lib for styling